.dockerignore

```diff
@@ -0,0 +1,17 @@
+.editorconfig
+.git
+.gitignore
+.github
+.idea
+.mvn
+.settings
+.vscode
+.project
+.travis.yml
+LICENSE
+README.md
+CHANGELOG.md
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+docker-compose.yml
+docs
```

Comment:
Add the following for faster frontend builds:
mico-admin/node #only used by maven
mico-admin/node_modules #npm install is already in dockerfile